zkstack_cli/crates/config/src/zkstack_config.rs

```diff
@@ -0,0 +1,27 @@
+use xshell::Shell;
+
+use crate::{ChainConfig, ChainConfigInternal, EcosystemConfig};
+
+pub enum ZkStackConfig {
+    EcosystemConfig(EcosystemConfig),
+    ChainConfig(ChainConfig),
+}
+
+impl ZkStackConfig {
+    fn from_file(shell: &Shell) -> anyhow::Result<ZkStackConfig> {
+        if let Ok(ecosystem) = EcosystemConfig::from_file(shell) {
+            Ok(ZkStackConfig::EcosystemConfig(ecosystem))
+        } else {
+            let chain_internal = ChainConfigInternal::from_file(shell)?;
+            let chain = ChainConfig::from_internal(chain_internal, shell.clone())?;
+            Ok(ZkStackConfig::ChainConfig(chain))
+        }
+    }
+
+    pub fn load_current_chain(shell: &Shell) -> anyhow::Result<ChainConfig> {
```

Comment:
let's name it just `current_chain` and also add a method for getting ecosystem

Reply:
[bf1f8da](https://github.com/matter-labs/zksync-era/pull/3199/commits/bf1f8da46320e9e4cc98ed9ff6d8d4a91601bf11) & [b2d7388](https://github.com/matter-labs/zksync-era/pull/3199/commits/b2d73882587fdd60e3505e397aeb1c13b89f3878)